src/main/resources/com/atlassian/bitbucket/jenkins/internal/trigger/BitbucketWebhookMultibranchTrigger/config.groovy

```diff
@@ -6,7 +6,7 @@ f.section() {
     f.entry(title: _("bitbucket.trigger.warning"), name: "warning", inline: "true") {}
 
     f.entry(title: _("bitbucket.trigger.refTrigger"), field: "refTrigger", name: "refTrigger", inline: "true") {
-        f.checkbox(checked:"true")
+        f.checkbox(checked:instance == null ? "true": instance.refTrigger)
```

Comment:
Commented on this in the other PR. I think this is more correct:
`f.checkbox(default:"true")`